custom_components/weenect/services.yaml

```diff
@@ -8,18 +8,20 @@ set_update_interval:
   fields:
     update_interval:
       required: true
-      example: "30M"
-      default: "30M"
+      example: "10M"
+      default: "10M"
       selector:
         select:
           options:
             - "0S"
             - "30S"
             - "1M"
+            - "2M"
+            - "3M"
             - "5M"
             - "10M"
             - "30M"
-            - "1H"
+            - "60M"
```

Comment:
Yeah I was confused by that as well. When I send 60M to the backend the next status update showed 1H. This was over a year ago though...